include/avif/avif.h

```diff
@@ -531,6 +531,18 @@ AVIF_API void avifImageStealPlanes(avifImage * dstImage, avifImage * srcImage, a
 //   * x/x/[1|2|5|6|9]/Limited
 //   * [1|2|5|6|9]/x/12/Limited
 
+// If libavif is built with libyuv fast paths enabled, libavif will use libyuv for conversion from
+// RGB to YUV if the following requirements are met:
+//
+// * YUV depth: 8
+// * RGB depth: 8
+// * rgb.chromaDownsampling: AVIF_CHROMA_UPSAMPLING_AUTOMATIC, AVIF_CHROMA_UPSAMPLING_FASTEST
+// * One of the following combinations (avifRGBFormat to avifPixelFormat/Range):
+//   *  BGRA            to [YUV400|YUV420|YUV422]/[Full|Limited]
+//   *  BGRA            to  YUV444               / Limited
+//   *  BGR             to  YUV420               /[Full|Limited]
+//   * [RGBA|ARGB|ABGR] to  YUV420               / Limited
```

Comment:
[Just a comment. No change requested.] It is difficult to verify this table matches the code in `avifImageRGBToYUVLibYUV8bpc()`.

If possible, we should reorder the code to make it more obvious that it matches this table in a follow-up pull request.

Reply:
Done at https://github.com/AOMediaCodec/libavif/pull/1009